cli/dts/lib.deno.ns.d.ts

```diff
@@ -2311,4 +2311,13 @@ declare namespace Deno {
     newpath: string,
     options?: SymlinkOptions,
   ): Promise<void>;
+
+  export interface MemoryUsage {
+    rss: number;
+    heapTotal: number;
+    heapUsed: number;
+    external: number;
+  }
+
+  export function memoryUsage(): MemoryUsage;
```

Comment:
I think these should go to `lib.deno_unstable.ns.d.ts` for now; I'd rather not add it as a stable API already

Reply:
I've moved the types to unstable